app/helpers/spree/admin/tax_categories_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Spree::Admin::TaxCategoriesHelper
+  def tax_category_dropdown_options(require_tax_category)
+    {
+      :include_blank => Spree::Config.products_require_tax_category ? false : t(:none), 
```

Comment:
I think we can use `require_tax_category` here in place of `Spree::Config.products_require_tax_category`